torch/csrc/jit/codegen/cuda/kernel_resource_strings.h

```diff
@@ -1,3 +1,4 @@
+#include "hip/hip_runtime.h"
```

Comment:
```suggestion
#ifdef __HIP_PLATFORM_HCC__
#include <hip/hip_runtime.h>
#endif
```
IIRC we put this include here due to the need to define certain types by this point, right?  Since this seemed only necessary for ROCm builds, and since you're including our specific HIP header explicitly, seems we should preprocessor protect it.

Reply:
I think I included this here by mistake. This is actually not needed. I have conditionally included this already in line 152 of kernel.cpp file. Will remove this.